dev/bench/bench.sh

```diff
@@ -237,6 +237,7 @@ On packages $coq_opam_packages
 fi
 
 zulip_edit() {
+    if ! [[ $zulip_post ]]; then return; fi
```

Comment:
Why not use `$ZULIP_BENCH_BOT` instead? If there is a transient failure in the first message sent to zulip, all subsequent messages will fail with this check.

Reply:
If there is a failure in sending the message there is nothing to edit.

Reply:
Right, my bad. I was under the wrong impression that the bot was posting new messages, not modifying the first one.